openproblems/tasks/dimensional_reduction/datasets/citeseq.py

```diff
@@ -0,0 +1,69 @@
+import os
+import tempfile
+
+import numpy as np
+import pandas as pd
+import scanpy as sc
+import scprep
+import anndata
+
+from ....data.utils import loader
+
+ADT_URL = "https://www.ncbi.nlm.nih.gov/geo/download/?acc=GSE100866&format=file&file=GSE100866%5FCBMC%5F8K%5F13AB%5F10X%2DADT%5Fumi%2Ecsv%2Egz"
+RNA_URL = "https://www.ncbi.nlm.nih.gov/geo/download/?acc=GSE100866&format=file&file=GSE100866%5FCBMC%5F8K%5F13AB%5F10X%2DRNA%5Fumi%2Ecsv%2Egz"
+
+
+@loader
+def load_citeseq_cbmc(test=False):
```

Comment:
This shouldn't reimplement the downloader -- just use he one defined in `openproblems.data.citeseq`. Then you can define a dataset function as in https://github.com/singlecellopenproblems/SingleCellOpenProblems/blob/master/openproblems/tasks/multimodal_data_integration/datasets/citeseq.py

Reply:
fixed thank you